### DISCUSSION TITLE
相同配置模式下，firefox和chrome进度条颜色不一

### DISCUSSION THREAD

**Discussion by silky-heart** [the1812/Bilibili-Evolved#3504] [提问 / Q&A]
chrome在开启夜间模式和自定义主题色后，进度条也变成自定义色，但是到了firefox（可能是版本102.0.1更新之后的事情）相同操作一遍，大部分按钮图标都能自定义色，除了进度条……

![QQ截图20220717081833](https://user-images.githubusercontent.com/76122458/179375875-0cf53a2e-67d8-41af-b97c-4b0cb1756524.jpg)

**Comment by the1812** [comment]
#3187
